packages/integration-tests/projects/suite-web/tests/suite/guide.test.ts

```diff
@@ -3,52 +3,65 @@
 
 describe('Test Guide', () => {
     beforeEach(() => {
-        cy.task('startEmu', { wipe: true });
-        cy.task('setupEmu');
-        cy.task('startBridge');
         cy.viewport(1024, 768).resetDb();
         cy.prefixedVisit('/');
-        cy.passThroughInitialRun();
     });
 
     it('Testing guide open / close', () => {
         // Open guide
         cy.getTestElement('@guide/button-open').click();
         cy.getTestElement('@guide/panel').should('be.visible');
+        cy.getTestElement('@guide/nodes')
+            .first()
+            .children()
+            .first()
+            .then(el => {
+                const text = el.text();
+                el.click();
+                cy.log('text', text);
+                cy.getTestElement('@guide/label').should('have.text', text);
+            });
+        cy.getTestElement('@guide/nodes').first().children().first().click();
+        cy.getTestElement('@guide/headerBreadcrumb/categoryLink').click();
```

Comment:
nit: the only `data-test` which is in camelCase

Reply:
changed